cli/cli.go

```diff
@@ -69,7 +69,9 @@ type Options struct {
 	TlsCA string
 	// Timeout is how long to wait for operations
 	Timeout time.Duration
-	// Username is the username or token to connect with
+	// ConnectionName is the name to use for the underlying NATS connection
+	ConnectionName string
+	// Nkey is the file holding a nkey to connect with
```

Comment:
Sorry I didn’t see this before but comments went wrong here. Other than that this is all good!